### ISSUE TITLE
Updated manuscript: copy_consensus analyses filtering update documentation

### ISSUE THREAD

**Issue by kgaonkar6** [AlexsLemonade/OpenPBTA-manuscript#132]
#### What section of the manuscript should be updated?

_Add the section (and subsection) needing an update._
Section: Somatic Copy Number Variant Calling (WGS samples only)
Sub section: Consensus CNV Calling

#### Proposed changes

_What changes need to be made and/or what details are missing?_
We need to add a few updates that were added in v19 and v20. These changes include 

1. adding controlfreec as default caller to add CN values which can be correctly compared to ploidy information
2. setting CN==NA for regions that were not called by any caller with CN alteration 
3. filtering Manta calls to only keep FILTER==PASS 
4. adding an update logic to include any CNV that overlaps a smaller CNV by 90% or more 

_Explain your rationale for the update._

1. [Add ControlFreec as default CN](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/964) since using cnvkit as default which when compared to ploidy from controlfreec added a lot of losses.
2. [Add Neutral CN==NA instead of original CN==2](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/1010) to remove the neutral calls instead of setting CN==2  for neutral because when comapred to a higher ploidy these neutral calls woul be coded as status==loss 
3. [Filter ==PASS forManta](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/1113) to remove low confidence SV calls
4. [Update overlap criteria](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/1125) which will add the common area of overlap if any CNV that overlaps 90% or more in any other CNV caller 

_Please include a level of detail that will allow another participant to make the update._
For points 1 and 2 above we looked into a subset of samples which show the benefits of setting neutral CN==NA and compared cnvkit with freec below:
https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/1010#issuecomment-832093409
We see that the changing neutral region CN to NA does infact help remove the losses seen all across the samples and genes. The rationale to use controlfreec as default rises from the fact that the ploidy is calculated by controlfreec and seems to have consistent calls if multiple samples exist for a sample

For points 3 we think it's reasonable to have the filter for PASS to keep only calls that manta coded as passing all caller specific filters.

For point 4 we were concerned that the 50% reciprocal calls is too stringent and we might thus miss out on calls that are in the multiple callers but not consistent in size of CNV. Which might arise from the caller specific limitations such that 1 caller calls a 2 smaller regions as CNV alteration instead of 1 large CNV called in another caller.

Example:
These are the raw calls
```
BS_K07KNTFY.cnvkit.dup.filtered3.bed:chr19	53641206	54049403	
BS_K07KNTFY.cnvkit.dup.filtered3.bed:chr19	54138551	54427104
BS_K07KNTFY.freec.dup.filtered3.bed:chr19	53641020	56141391
```

and this is the output , which covers the common areas
```r
# A tibble: 2 x 7
  ID          chrom loc.start  loc.end num.mark seg.mean copy.num
  <chr>       <chr>     <dbl>    <dbl> <lgl>       <dbl>    <dbl>
1 BS_K07KNTFY chr19  53641206 54049403 NA           1.27       12
2 BS_K07KNTFY chr19  54138551 54427104 NA           1.36       12

#### Relevant literature

_If there is relevant scientific literature, put links to those items here._

**Comment by jharenza** [comment]
@runjin326 will work on this

**Comment by jaclyn-taroni** [comment]
Closed via #145
